replication-api/src/main/java/com/connexta/replication/api/data/Persistable.java

```diff
@@ -22,18 +22,4 @@
    * @return The ID of the object
    */
   String getId();
-
-  /**
-   * Sets the ID of the persistable object
-   *
-   * @param id the ID to set as the persistable object's ID
-   */
-  void setId(String id);
-
-  /**
-   * Returns the version of this persistable
-   *
-   * @return integer version of the persistable
-   */
-  int getVersion();
```

Comment:
❓ This was remove because the object always represents the latest version, right?

Reply:
`getVersion()` is for the schema version and not the object version 😉